README.md

```diff
@@ -116,6 +116,7 @@ Activities require a certain formatting:
   durationHours, // a floating point that represents the duration of the activity in decimal hours
   activityType: ACTIVITY_TYPE_LODGING,
   hotelClass, // a variable (from definitions.js) that represents the class of the hotel
+  country, // a variable (from definitions.js) that represents the country of the hotel
```

Comment:
should be `countryCodeISO2` instead (to make it extremely clear we're using the ISO2 code)